### PR TITLE
Update configuration location

### PR DESCRIPTION
**Pull request by shijie666**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by dengliming** [comment]
Thanks for your contribution. I think it's duplicate with https://github.com/dromara/soul/pull/1040

File: soul-bootstrap/src/main/resources/application-local.yml [hunk 1]
```diff
@@ -42,26 +42,25 @@ soul:
...
+#          url: localhost:8848
+#          namespace: 1c10d748-af86-43b9-8265-75f487d20c6c
+#          acm:
+#            enabled: false
+#            endpoint: acm.aliyun.com
+#            namespace:
+#            accessKey:
+#            secretKey:
+    exclude:
+      enabled: false
```
Comment (yu199195) [inline soul-bootstrap/src/main/resources/application-local.yml:61]: what mean？

**Review by yu199195 — DISMISSED** [review]

File: soul-bootstrap/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ soul:
         websocket :
              urls: ws://localhost:9095/websocket
     exclude:
-      enable: false
+      enabled: false
```
Comment (yu199195) [inline soul-bootstrap/src/main/resources/application-local.yml:46]: is this enabled？  can you move this config？

Comment (shijie666) [reply soul-bootstrap/src/main/resources/application-local.yml]: Has moved, Conflict resolved

**Review by yu199195 — APPROVED** [review]

**Pull request merged**